klippy/toolhead.py

```diff
@@ -457,13 +457,14 @@ def _flush_handler(self, eventtime):
     # Movement commands
     def get_position(self):
         return list(self.commanded_pos)
-    def set_position(self, newpos, homing_axes=()):
+    def set_position(self, newpos, homing_axes=(), clear_axes=()):
```

Comment:
I don't think we should add a `clear_axes` parameter to `set_position()`.  Instead, force_move can explicitly call `toolhead.get_kinematics().clear_homing_state()` (as save_z_home does).